Cargo.toml

```diff
@@ -81,7 +81,7 @@ engine_rocks = { path = "components/engine_rocks" }
 engine_traits = { path = "components/engine_traits" }
 error_code = { path = "components/error_code" }
 fail = "0.3"
-raft-engine = { git = "https://github.com/tikv/raft-engine", default-features = false }
+raft-engine = { git = "https://github.com/tikv/raft-engine"}
```

Comment:
I think tikv could depend `component/raft_log_engine` and not need to depend `raft-engine` directly.